airbyte-integrations/connectors/destination-snowflake/src/test-integration/kotlin/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestinationAcceptanceTest.kt

```diff
@@ -290,7 +290,7 @@ open class SnowflakeInsertDestinationAcceptanceTest : DestinationAcceptanceTest(
         private val NAME_TRANSFORMER: NamingConventionTransformer = SnowflakeSQLNameTransformer()
         @JvmStatic
         protected val NO_ACTIVE_WAREHOUSE_ERR_MSG: String =
-            "No active warehouse selected in the current session.  Select an active warehouse with the 'use warehouse' command."
+            "No active warehouse selected in the current session or insufficient permissions.  Select a viable warehouse  with the 'use warehouse' command."
```

Comment:
note: you changed the copy in the test, not the implementation

Reply:
@evantahler oof that's my bad. The problem is that it looks like it gets reported within a catch clause and that specific error message is just coming from the driver. So I guess the only way to modify that message for airbyte is to modify that first part "Could not connect with provided configuration":

`                .withMessage("Could not connect with provided configuration. ${e.message}")`

Not sure if that catch message will always be the same, if it's different then modifying the first part might stop making sense here.

Reply:
I think this might be something to wait on.  It's probably best not to modify the message we get from the driver at this level, but instead in our new "error translation framework" we are rolling out.  We are rolling it out for sources first (e.g. https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-postgres/src/main/kotlin/io/airbyte/integrations/source/postgres/PostgresSourceExceptionHandler.kt), but it allows to match errors with regular expressions and then 'translate' them.

Reply:
We'll start brining this into destinations in the next few months.